### PR TITLE
RHEL 9.4 for SAP build configs.

### PR DESCRIPTION
**Pull request by zmarano**

### CODE REVIEW

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel_9_4_sap.cfg [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+cp /run/install/isodir/65-gce-disk-naming.rules /etc/udev/rules.d/
+cp /run/install/isodir/google_nvme_id /usr/lib/udev/
+chmod +x /usr/lib/udev/google_nvme_id
+# Wait for coldplug events from boot to settle, or we won't generate new events for the reload/trigger
+udevadm settle
+udevadm control --reload
+udevadm trigger --settle
+tee -a /tmp/disk-config << EOM
+# build_installer.py will replace with the id of the install disk to avoid race conditions
+bootloader --boot-drive=/dev/disk/by-id/google-el-install-disk --timeout=0 --append="net.ifnames=0 biosdevname=0 scsi_mod.use_blk_mq=Y"
```
Comment (a-crate) [inline daisy_workflows/image_build/enterprise_linux/kickstart/rhel_9_4_sap.cfg:33]: Don't disable persistent names

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel_9_4_sap_byos.cfg [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+cp /run/install/isodir/65-gce-disk-naming.rules /etc/udev/rules.d/
+cp /run/install/isodir/google_nvme_id /usr/lib/udev/
+chmod +x /usr/lib/udev/google_nvme_id
+# Wait for coldplug events from boot to settle, or we won't generate new events for the reload/trigger
+udevadm settle
+udevadm control --reload
+udevadm trigger --settle
+tee -a /tmp/disk-config << EOM
+# build_installer.py will replace with the id of the install disk to avoid race conditions
+bootloader --boot-drive=/dev/disk/by-id/google-el-install-disk --timeout=0 --append="net.ifnames=0 biosdevname=0 scsi_mod.use_blk_mq=Y"
```
Comment (a-crate) [inline daisy_workflows/image_build/enterprise_linux/kickstart/rhel_9_4_sap_byos.cfg:33]: Persistent names

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel_9_4_sap.cfg (hunk 1 above)
Comment (zmarano) [reply daisy_workflows/image_build/enterprise_linux/kickstart/rhel_9_4_sap.cfg]: Done. Good call that we want to move this way going forward.

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel_9_4_sap_byos.cfg (hunk 2 above)
Comment (zmarano) [reply daisy_workflows/image_build/enterprise_linux/kickstart/rhel_9_4_sap_byos.cfg]: Done

**Comment by a-crate** [comment]
You have an trailing `"` on the bootloader line, there's no open quote for it (timeout is unquoted)

**Comment by zmarano** [comment]
> You have an trailing `"` on the bootloader line, there's no open quote for it (timeout is unquoted)

Whoops. Fixed.

**Review by a-crate — APPROVED** [review]

**Pull request merged**